src/components/CreatureModal.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import type { Pokemon } from '../types';
+import './CreatureModal.css';
+
+type CreatureCardModalProps = {
+  pokemon: Pokemon;
+};
+
+export default function CreatureCardModal({ pokemon }: CreatureCardModalProps) {
+  return (
+    <div className="card">
+      <img src={pokemon.image ?? ''} alt={`${pokemon.name}`} />
+      <p>#{pokemon.id}</p>
+      <ul>
+        <li><p>Name: {pokemon.name}</p></li>
+        <li>
+          <p>Type/s:</p>
+          <ul>
+            {pokemon.types.map((type, idx) => (
+              <li key={idx}>{type}</li>
```

Comment:
Is a good practice create an unique key. Combine the index with the name or the pokemon index. Ex: `{`${pokemon_name}_${idx}`}`